Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<ol><li>In order to iterate over the pairs <code>(l[i], l[i+1])</code>  ,  use <code>zip(l[:-1], l[1:])</code> In the example above - iterate over pairs of <code> seasons</code> . This also makes <code>enumerate</code>  redundant.</li><li>Where did you see that the bottom 3 teams go down one league ? Is there a way to validate this assumption using only the data (i.e., without using the internet) ?</li><li> Please use a straight-forward syntax for calculating the difference between two sets: <code>a - b</code>  </li><li>Add an explicit string explaining the meaning of the printed pairs.</li><li>Make assumptions explicit: This piece of code assumes that the teams are sorted according their score in the season. Please include the sorting command of the teams in this cell. </li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>2.Not realy, becuse we sow that not all the team that leave the league and one season just gone it's becuse they need to down one league.</p><p>3.why? it's the same.</p>

Reply:
<p>2. There must be a way to validate this assumption. Please think it over and try looking into the data.</p><p>3. Because it is more concise and beautiful. See first line in the zen</p><p>https://www.python.org/dev/peps/pep-0020/#the-zen-of-python</p>